### PR TITLE
Feature/local cob4 changes

### PR DESCRIPTION
**Pull request by benmaidel**
local changes from supplier account on cob4-2. Not sure what is needed what not

### CODE REVIEW

File: cob_hardware_config/cob4-2/config/flexisoft.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 host: 10.4.2.99
 port: "9100" # alternative syntax: !!str 9100
 odometry_timeout: 1.0
-threshold_linear_slow: 0.1
-threshold_linear_fast: 0.4
-threshold_angular_fast: 0.5
+threshold_linear_slow: 1.1
```
Comment (NadiaHG) [inline cob_hardware_config/cob4-2/config/flexisoft.yaml:4]: @ipa-tif I think that you suggested these values, make sense keep them or we should revert the changes?

File: cob_bringup/drivers/softkinetic.launch [hunk 2]
```diff
@@ -6,15 +6,27 @@ The arguments given are the device indices of the cameras determined by the Dept
...
+    <param name="min_neighbours" type="int" value="50" />
+    <param name="depth_mode" value="close" />  <!-- close or long -->
+    <param name="depth_frame_format" value="QVGA" /> <!-- ignored -->
+    <param name="depth_frame_rate" value="25" /> <!-- 25, 30, 50, 60 -->
+    <param name="color_compression" value="MJPEG" /> <!-- YUY2, MJPEG -->
+    <param name="color_frame_format" value="WXGA_H" /> <!-- QQVGA, QVGA, VGA, WXGA_H, NHD -->
+    <param name="color_frame_rate" value="25" /> <!-- 25, 30 -->
+  </node>
+
+  <node name="cam3d_down_relay_depth_points" pkg="topic_tools" type="relay" args="/$(arg name)/depth/points /$(arg name)/depth_registered/points" respawn="false" output="screen"/>
```
Comment (NadiaHG) [inline cob_bringup/drivers/softkinetic.launch:29]: @fmw-hb why do you have to relay from ../depth/points to ../depth_registered/points?

File: cob_hardware_config/cob4-2/config/flexisoft.yaml (hunk 1 above)
Comment (floweisshardt) [inline cob_hardware_config/cob4-2/config/flexisoft.yaml:4]: we should keep the original values with `0.x`. we just increased the values for MSH testing only. Please revert.

File: cob_bringup/drivers/softkinetic.launch (hunk 2 above)
Comment (floweisshardt) [inline cob_bringup/drivers/softkinetic.launch:29]: should not be a separate node (will double network load!!). Please use normal remapping within the node or launch tag.

File: cob_hardware_config/cob4-2/config/battery_monitor.yaml [hunk 3]
```diff
@@ -2,9 +2,9 @@ threshold_warning: 20 # % of battery level
 threshold_error: 10 # % of battery level
 threshold_critical: 5 # % of battery level
 
-enable_light: true
+enable_light: false
```
Comment (floweisshardt) [inline cob_hardware_config/cob4-2/config/battery_monitor.yaml:5]: why did you disable light and sound?

File: cob_hardware_config/cob4-1/urdf/calibration_default.urdf.xacro [hunk 4]
```diff
@@ -80,9 +80,9 @@
 	<property name="def_torso_cam3d_down_x" value="0.15908"/>
 	<property name="def_torso_cam3d_down_y" value="0.0"/>
 	<property name="def_torso_cam3d_down_z" value="0.14005"/>
-	<property name="def_torso_cam3d_down_roll" value="2.18166"/> <!--2.18166-->
-	<property name="def_torso_cam3d_down_pitch" value="0.0"/>
-	<property name="def_torso_cam3d_down_yaw" value="1.57079"/> <!--1.57079-->
+	<property name="def_torso_cam3d_down_roll" value="0.0"/> <!--2.18166-->
```
Comment (floweisshardt) [inline cob_hardware_config/cob4-1/urdf/calibration_default.urdf.xacro:83]: where do these values come from?

**Comment by floweisshardt** [comment]
I'd rather close this PR and separate usefull parts (softkinetic) into a new PR

File: cob_hardware_config/cob4-2/config/battery_monitor.yaml (hunk 3 above)
Comment (benmaidel) [inline cob_hardware_config/cob4-2/config/battery_monitor.yaml:5]: sound and light is totally annoying because bms always says that the battery is empty. And we do not want to have it in the msh scenario.

File: cob_hardware_config/cob4-1/urdf/calibration_default.urdf.xacro (hunk 4 above)
Comment (benmaidel) [inline cob_hardware_config/cob4-1/urdf/calibration_default.urdf.xacro:83]: i have no clue. This was just changed and uncommited on the suplier account on the cob4-2.
@ipa-nhg @fmw-hb ?

File: cob_hardware_config/cob4-2/config/flexisoft.yaml (hunk 1 above)
Comment (benmaidel) [inline cob_hardware_config/cob4-2/config/flexisoft.yaml:4]: which values do we need if we are at saturn? They've also been changed inside the msh account, from where we've always started our robot.launch. So @ipa-fmw are you sure we should revert this?

File: cob_hardware_config/cob4-1/urdf/calibration_default.urdf.xacro (hunk 4 above)
Comment (NadiaHG) [inline cob_hardware_config/cob4-1/urdf/calibration_default.urdf.xacro:83]: The values are not needed, at least for cob4-1, this robot will get a realsense camera. 
The softkinetic calibration can be tested on cob4-5, as soon as the hardware is ready

File: cob_bringup/drivers/softkinetic.launch (hunk 2 above)
Comment (benmaidel) [inline cob_bringup/drivers/softkinetic.launch:29]: right, if there is a node who is expecting `depth_registered/points` from softkinetic, they have to remap their input topics

**Comment by benmaidel** [comment]
I've created separate PR's for this changes. There is just the open question concerning the flexisoft params.

File: cob_hardware_config/cob4-2/config/flexisoft.yaml (hunk 1 above)
Comment (floweisshardt) [inline cob_hardware_config/cob4-2/config/flexisoft.yaml:4]: we currently need to shrink the safety fields down to a minimum so that they are within the navigation's footprint. But this setup is not good for running 24/7 tests within MSH.

In order to use the desired safety fields which are selected based on the current velocity of the robot we need to find a way that the navigation can deal with changing footprints. --> @ipa-srd: do you have a solution ready?

Comment (floweisshardt) [inline cob_hardware_config/cob4-2/config/flexisoft.yaml:4]: @ipa-uhr: we should put that on the agenda for 24/7 tests in the store

**Comment by floweisshardt** [comment]
see my comments above regarding flexisoft params

**Comment by benmaidel** [comment]
closing this, because it is not needed anymore!
